### PR TITLE
MAT-6568: Change label to conditional formatting AND styling for required text

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-6568](https://jira.cms.gov/browse/MAT-6568)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx [hunk 1]
```diff
@@ -111,7 +121,7 @@ const MultipleSelectDropDown = ({
           inputProps["aria-label"] = "Types multiple types can be selected";
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: I think this should be `inputProps["aria-label"] ="Measure types multiple measure types can be selected"`

Comment (adongare) [reply src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: there is  inputProps["aria-describedby"] = "measure-group-type-helper-text"; prop on this which is not visible here but I think is not required.

File: src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx [hunk 2]
```diff
@@ -56,6 +56,16 @@ const MultipleSelectDropDown = ({
   onClose,
   ...rest
 }) => {
+  const requiredLabelReadable = (
+    <span style={{ color: "white", fontSize: "1px" }}>required</span>
+  );
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: can we use this instead` const requiredLabelReadable = <span className="sr-only">required</span>;`

Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: Are we able to use `className` prop in our app ? I need to revisit tailwind and twin.macro setup once again, I forgot. but anyways good to know that Tailwind has this class.

Comment (adongare) [reply src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: I think we have all sorts of CSS styling approaches in our app not just tailwind. I don't think `sr-only` is from tailwind. It is from design systems

Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: ooh There is sr-only in Tailwind as well, but yeah I see your point then, Even className should work fine.

Comment (ethankaplan) [reply src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx]: I didn't know about this!

**Review by adongare — APPROVED** [review]

**Pull request merged**